components/private/UIMetrics/src/MDCLayoutMetrics.h

```diff
@@ -15,6 +15,8 @@
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
 
+extern const CGFloat kFixedStatusBarHeightOnPreiPhoneXDevices;
+
```

Comment:
Exposing this constant in the header here hopefully will make it more straightforward for other components to move off of `MDCDeviceTopSafeAreaInset` in a similar way to how is being done the bottom drawer. However, if desired I'm happy to duplicate this constant within the bottom drawer and not modify `MDCLayoutMetrics`.